### DISCUSSION TITLE
Firmware of the platform via the Eclipse IDE for C and C++

### DISCUSSION THREAD

**Discussion by sanjokbu** [grblHAL/core#425] [Q&A] [answered]
Hello friends.

I ask for your help

I installed the program "Eclipse IDE for C and C++", did everything according to this instruction: https://github.com/grblHAL/core/wiki/Compiling-Grilhala

When you click the "Build project" button, it displays the inscription "Nothing to build for project"

What am I doing wrong and where can I find a solution to the problem?

![1](https://github.com/grblHAL/core/assets/156105565/2241a59c-d230-4ae8-92d8-43e2b1b43b55)

**Comment by terjeio** [comment, marked as answer]
I don't know how easy it will be to set up the generic Eclipse for compilation. I am using [MCUExpresso](https://www.nxp.com/design/design-center/software/development-software/mcuxpresso-software-and-tools-/mcuxpresso-integrated-development-environment-ide:MCUXpresso-IDE) which has the needed tools preconfigured. Currently I am using v11.3.1.

**Reply by sanjokbu** [reply]
спасибо большое)
